### PR TITLE
Update cache size

### PR DESCRIPTION
**Pull request by nagarev**
## Description

Increase cache size

## Motivation and Context

Mr. Angel Lopez recommended this value to be upgraded to improve node execution times, a few tests where made to confirm that.

## How Has This Been Tested?

Using Unit Tests
Profiling Tool

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Requires Activation Code (Hard Fork)

* **Other information**:

### CODE REVIEW

File: rskj-core/src/main/resources/reference.conf [hunk 1]
```diff
@@ -405,7 +405,7 @@ cache {
     # Processing 134 blocks grows the commitedCache to 100K entries, or approximately
     # or 25 Mbytes. A cache of about 100 Mbytes seems rasonable. Anyway, for
     # precaution we'll set the limit to 100K entries.
-    max-elements: 100000
+    max-elements: 1000000
```
Comment (marcciosilva) [inline rskj-core/src/main/resources/reference.conf:408]: we should update the comments preceding this value, right?

**Comment by marcciosilva** [comment]
pipeline:run

File: rskj-core/src/main/resources/reference.conf (hunk 1 above)
Comment (nagarev) [reply rskj-core/src/main/resources/reference.conf:409]: Done!

**Review by marcciosilva — APPROVED** [review]
lgtm!

**Comment by nagarev** [comment]
pipeline:run

**Pull request merged**